com.redhat.patchpal.gui.yaml

```diff
@@ -0,0 +1,38 @@
+app-id: com.redhat.patchpal.gui
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.llvm20
+command: patchpal
+separate-locales: false
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-config
+
+modules:
+  - git.json
```

Comment:
this should be inline. seperate manifest, unless autogenerated increase the maintenance burden.

Reply:
It's copied from another project:
https://github.com/flathub/com.github.git_cola.git-cola/blob/master/git.json

Reply:
that doesn't make it right.

Reply:
Isn't "copying from a well-known source that tracks upstream" equivalent to "autogenerated"? I'm trying to understand how fetching a file is more time-consuming and therefore a maintenance burden than copy/pasting and reindenting code.